docs/book/v3/application-integration/usage-in-a-laminas-mvc-application.md

```diff
@@ -17,19 +17,11 @@ To use a session container some configuration for the component is needed:
 - a [storage handler](../storage.md)
 - some [configuration for the session](../config.md) itself
 
-To allow a [reflection-based approach](https://docs.laminas.dev/laminas-servicemanager/reflection-abstract-factory/)
-to retrieve the session container from the service manager, a class name is
-needed as the name for the container. The example uses the name
-`Laminas\Session\Container::class`.
-
 Add the following lines to the local or global configuration file, e.g.
 `config/autoload/global.config.php`:
 
 ```php
 return [
-    'session_containers' => [
-        Laminas\Session\Container::class,
-    ],
```

Comment:
Deleting without a replacement makes the tutorial useless!

Reply:
this example use the `Laminas\Session\Container` which was newly defined in the ConfigProvider factory list. No need to have a AbstractFactory and session_containers config for getting the default container.

Or do you want that i update this documentation to use both ways? The default Container like this without any configuration steps and the other one like in the v2 to v3 Migration guide Documentation?